bitnami/mean/requirements.yaml

```diff
@@ -3,6 +3,3 @@ dependencies:
   repository: https://kubernetes-charts.storage.googleapis.com/
   version: 4.x.x
   condition: mongodb.install
-- name: bitnami-common
```

Comment:
why are you deleting this? This chart is required if we want to deploy cloud resources using k8s service-catalog